crypto/asn1/standard_methods.h

```diff
@@ -30,6 +30,9 @@ static const EVP_PKEY_ASN1_METHOD *standard_methods[] = {
 #ifndef OPENSSL_NO_CMAC
     &cmac_asn1_meth,
 #endif
+#ifndef OPENSSL_NO_RSA
+    &rsa_pss_asn1_meth,
+#endif
```

Comment:
It means they should be kept in ascending order of the type (i.e. NID) each method represents which corresponds to the pkey_id field of EVP_PKEY_ASN1_METHOD. This is needed because we use OBJ_bsearch() to lookup methods by pkey_id. The comment could make that clearer.

BTW if the table  gets out of order then there is a test (pkey_meth_test) which will pick it up.